doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-system-properties-ref.adoc

```diff
@@ -0,0 +1,361 @@
+
+|`org.kie.executor.pool.size`
+|Integer
+|`1`
+|The number of threads used by the {PRODUCT} executor.
+
+|`org.kie.executor.retry.count`
+|Integer
+|`3`
+|The number of retries the {PRODUCT} executor attempts on a failed job.
+
+|`org.kie.executor.jms.queue`
+|String
+|`queue/KIE.SERVER.EXECUTOR`
+|Job executor JMS queue for {KIE_SERVER}.
+
+|`org.kie.executor.disabled`
+|`true`, `false`
+|`false`
+|If set to `true`, disables the {KIE_SERVER} executor.
+|===
+
+.Human task system properties
+[cols="2,1a,1,3a", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`org.jbpm.ht.callback`
+|`mvel`
+
+`ldap`
+
+`db`
+
+`jaas`
+
+`props`
+
+`custom`
+|`jaas`
+|A property that specifies the implementation of user group callback to be used:
+
+* `mvel`: Default; mostly used for testing.
+* `ldap`: LDAP; requires additional configuration in the `jbpm.usergroup.callback.properties` file.
+* `db`: Database; requires additional configuration in the `jbpm.usergroup.callback.properties` file.
+* `jaas`: JAAS; delegates to the container to fetch information about user data.
+* `props`: A simple property file; requires additional file that will keep all information (users and groups).
+* `custom`: A custom implementation; specify the fully qualified name of the class in the `org.jbpm.ht.custom.callback` property.
+
+|`org.jbpm.ht.custom.callback`
+|Fully qualified name
+|N/A
+|A custom implementation of the `UserGroupCallback` interface in case the `org.jbpm.ht.callback` property is set to `custom`.
+
+|`org.jbpm.task.cleanup.enabled`
+|`true`, `false`
+|`true`
+|Enables task cleanup job listener to remove tasks once the process instance is completed.
+
+|`org.jbpm.task.bam.enabled`
+|`true`, `false`
+|`true`
+|Enables task BAM module to store task related information.
+
+|`org.jbpm.ht.admin.user`
+|String
+|`Administrator`
+|User who can access all the tasks from {KIE_SERVER}.
+
+|`org.jbpm.ht.admin.group`
+|String
+|`Administrators`
+|In order to display all the tasks from {KIE_SERVER}, users must belong to the `Administrators` group.
+|===
+endif::[]
+
+.System properties for loading keystore
+[cols="2,1,1,3", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`kie.keystore.keyStoreURL`
+|URL
+|N/A
+|URL is used to load a Java Cryptography Extension KeyStore (JCEKS). For example, `file:///home/kie/keystores/keystore.jceks`.
+
+|`kie.keystore.keyStorePwd`
+|String
+|N/A
+|The password is used for the JCEKS.
+
+|`kie.keystore.key.server.alias`
+|String
+|N/A
+|The alias name of the key for REST services where the password is stored.
+
+|`kie.keystore.key.server.pwd`
+|String
+|N/A
+|The password of an alias for REST services.
+
+|`kie.keystore.key.ctrl.alias`
+|String
+|N/A
+|The alias of the key for default REST {CONTROLLER}.
+
+|`kie.keystore.key.ctrl.pwd`
+|String
+|N/A
+|The password of an alias for default REST {CONTROLLER}.
+|===
+
+.Other system properties
```

Comment:
Just a heads-up that this table is over expanded a bit in community. I think it has to do with the columns set as "2,1,1,3a" instead of "2,1,1,3". I typically just use percentage, for what it's worth, like "30%,20%,20%,30%" or something.

Not critical, just if it's an obvious fix.